recipe/meta.yaml

```diff
@@ -24,38 +28,35 @@ build:
     - MIGRATING={{ migrating }}
 
 requirements:
-  {% if not win %}
-  build:
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - ipython >=7.23.1                       # [build_platform != target_platform]
-    - jupyter_client >=6.1.12                # [build_platform != target_platform]
-    - packaging                              # [build_platform != target_platform]
-    - psutil                                 # [build_platform != target_platform]
-    - python                                 # [build_platform != target_platform]
-  {% endif %}
   host:
     - debugpy >=1.0.0
     - ipython >=7.23.1
     - jupyter_client >=6.1.12
     - packaging
     - pip
     - psutil
-    - python
+    - python 3.9
   run:
-    - appnope  # [osx]
+    {% if noarch_platform == "unix" %}
+    - __unix
+    - appnope
```

Comment:
Is `appnope` used on Linux? If not, would it make sense to have this be `osx`?

Reply:
added osx variant